nuxeo-core/nuxeo-core-storage-mongodb/src/main/java/org/nuxeo/ecm/core/storage/mongodb/MongoDBChecker.java

```diff
@@ -66,4 +82,20 @@ public void check(ConfigurationGenerator cg) throws ConfigurationException {
             ret.close();
         }
     }
+
+    /**
+     * Returns the value of the check timeout parameter in seconds.
+     * If value is not parseable or not set, then use de default value.
+     * @return the timeout check in seconds.
+     * @since 9.3
+     */
+    private int getCheckTimeoutInSeconds(ConfigurationGenerator cg) {
+        int checkTimeout = DEFAULT_CHECK_TIMEOUT_IN_SECONDS;
+        try {
+            checkTimeout = Integer.parseInt(cg.getUserConfig().getProperty(PARAM_MONGODB_CHECK_TIMEOUT, DEFAULT_CHECK_TIMEOUT_IN_SECONDS.toString()));
```

Comment:
`DEFAULT_CHECK_TIMEOUT_IN_SECONDS.toString()` -> `String.valueOf(DEFAULT_CHECK_TIMEOUT_IN_SECONDS)` now that it's an `int`